### PR TITLE
adding cash allocation to dto

### PR DESCRIPTION
**Pull request by antoinemeyer**
https://trizic.atlassian.net/browse/TRIZ-3461

@johntowell @abuecker @LowellBateman please review

### CODE REVIEW

File: api/v2/schema/model-portfolio.res.json [hunk 1]
```diff
@@ -34,7 +37,7 @@
           "type": "string"
         },
         "percentage": {
-          "type": "integer"
+          "type": "number"
```
Comment (antoinemeyer) [inline api/v2/schema/model-portfolio.res.json]: also changed that to number

File: api/v2/schema/model-portfolio.res.json [hunk 2]
```diff
@@ -24,6 +24,9 @@
       "description": "The allocations for the model portfolio",
       "type": "array",
       "items": { "$ref": "#/definitions/assetAllocation" }
+    },
```
Comment (johntowell) [inline api/v2/schema/model-portfolio.res.json]: add this to the required block below for the schema to actually be useful

Comment (antoinemeyer) [inline api/v2/schema/model-portfolio.res.json]: added

**Comment by johntowell** [comment]
:+1: sorry for delay

**Comment by antoinemeyer** [comment]
I need 2 :+1: for that, who volunteers? 
@eweise @bondricek @abuecker @LowellBateman @eselman @pablodicosta

**Comment by bondricek** [comment]
:+1:

**Comment by antoinemeyer** [comment]
thanks @bondricek

**Pull request merged**